README.md

```diff
@@ -71,7 +71,7 @@ To test against a different version of Emacs, set the environment variable `EMAC
 
 ## Alternatives
 
-- Tree-sitter in Emacs's core: https://github.com/casouri/emacs (the future)
+- Tree-sitter in Emacs's core: https://github.com/emacs-mirror/emacs/tree/feature/tree-sitter (the future)
```

Comment:
I think it's also worth replacing `(the future)` with `(Emacs 29+)`.

Reply:
Oh, and I just realized, since it's now on master, probably worth replacing link to https://github.com/emacs-mirror/emacs/blob/master/lisp/treesit.el